tilebox-datasets/tilebox/datasets/query/time_interval.py

```diff
@@ -8,7 +8,7 @@
 from google.protobuf.timestamp_pb2 import Timestamp
```

Comment:
Moving this is a breaking change - e.g. we have examples in the docs using the old import here:

https://docs.tilebox.com/datasets/query/filter-by-time#endpoint-inclusivity

Let's do the following:

Create `tilebox.datasets.data.time_interval.py` again - and let it import `TimeInterval` from `tilebox.datasets.query.time_interval.TimeInterval` instead.

And let's also emit a deprecation warning in the module telling people to please update their import statements.

then finally, fix the occurences in the docs.